index.js

```diff
@@ -345,6 +345,7 @@ module.exports = function (log, indexesPath) {
   function getSeqFromOffset(offset) {
     if (offset === -1) return -1
     const { tarr, count } = seqIndex
+    if (tarr[count - 1] === offset) return count - 1
```

Comment:
This here is just an optimization to avoid the binary search. If all indexes are up to date, then we avoid the binary search and we can just peek at the end of the seq `tarr`.